taxpose/datasets/ndf.py

```diff
@@ -78,6 +74,13 @@ class NDFPointCloudDatasetConfig:
     action: Phase = Phase.GRASP
     symmetry_after_transform: bool = False
 
+    # Augmentation parameters.
+    occlusion_cfg: Optional[OcclusionConfig] = None
+    synthetic_occlusion: bool = False
```

Comment:
nit: cant you just check for existence of occlusion_cfg?

Reply:
added